src/containers/BooksList.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import Book from '../components/Book';
+
+function BooksList({
+  books,
+}) {
+  return (
+    <section>
+      {
+          books.length > 0
+            ? books.map((book) => (
+              <Book
+                key={book.id}
+                book={book}
+              />
+            ))
+            : <h3 style={{ textAlign: 'center' }}>No books were found!</h3>
+        }
+    </section>
```

Comment:
Please change your component to display books in an __HTML table__ using the `Book` component ([screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/BooksAsTable.png))

Reply:
This is resolved well